clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_logging-app_rbac.yaml

```diff
@@ -6,3 +6,39 @@ metadata:
     openshift.io/description: This SA is used for uploading audit-log by vector
   name: logging-app
   namespace: api-audit-log
+---
+apiVersion: security.openshift.io/v1
```

Comment:
Why do we need this now, the manifest is unchanged?

Reply:
Nope... that was always needed. I don't have any clue how it worked before. I guess we applied the scc manually?

Reply:
So it already existed and just wasn't in git?

Reply:
Yes. Here I add a specific scc for the `logging-app` service account.